.github/workflows/runtime-image-update.yaml

```
@@ -0,0 +1,105 @@
+---
+name: Update runtime image tag
+
+on:  # yamllint disable-line rule:truthy
+  schedule:
+    - cron: '0 0 * * *'  # Run every day at midnight UTC
+  workflow_dispatch:
+    inputs:
+      quay_repo:
+        description: 'Quay Repository'
+        default: 'astronomer/astro-runtime'
+
+jobs:
+  runtime-image-tag-update-job:
+    runs-on: 'ubuntu-20.04'
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.9'
+
+      - name: Install dependencies
+        run: pip install requests semantic-version
+
+      - name: Get current tag from Dockerfile.astro_cloud
+        id: get_current_tag
+        # yamllint disable rule:line-length
+        run: |
+          CURRENT_TAG=$(awk -F: '/quay.io\/astronomer\/astro-runtime/ {print $NF}' ./.circleci/integration-tests/Dockerfile.astro_cloud)
+          echo "CURRENT_TAG=$CURRENT_TAG"
+          echo "::set-output name=tag::$CURRENT_TAG"
```

This seems to be deprecating.
https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/

```suggestion
          echo "tag=$CURRNET_TAG" >> $GITHUB_OUTPUT
```

---

same applied to elsewhere

---

Thank you for this. Created a PR to address this https://github.com/astronomer/astronomer-providers/pull/1163